azkaban-common/src/main/java/azkaban/executor/ExecutorApiClient.java

```diff
@@ -31,6 +51,63 @@
 @Singleton
 public class ExecutorApiClient extends RestfulApiClient<String> {
 
+  final private static Logger logger = Logger.getLogger(ExecutorApiClient.class);
+
+  final private String DEFAULT_TRUSTSTORE_PATH = "keystore";
+  final private String DEFAULT_TRUSTSTORE_PASSWORD = "changeit";
+  final private boolean isTlsEnabled;
+  final private String truststorePath;
+  final private String truststorePassword;
+  private SSLConnectionSocketFactory tlsSocketFactory;
+
+  @Inject
+  public ExecutorApiClient(final Props azkProps) {
+    this.isTlsEnabled = azkProps.getBoolean(EXECUTOR_CONNECTION_TLS_ENABLED, false);
+    this.truststorePath = azkProps
+        .getString(Constants.JETTY_TRUSTSTORE_PATH, this.DEFAULT_TRUSTSTORE_PATH);
```

Comment:
There is already a property name azkaban.csr.truststore.location for truststore location. Will it be possible to reuse it here?